zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -421,38 +406,11 @@ fn storage_key_mismatch() {
         tx_id: 4,
         committed_value: U256::from(34),
     }];
-    let overrides = HashMap::from([((AdviceColumn::StorageKeyByte1, 0), Fr::ONE)]);
+    let overrides = HashMap::from([((AdviceColumn::StorageKeyLimb0, 0), Fr::ONE)]);
 
     let result = verify_with_overrides(rows, overrides);
 
-    assert_error_matches(result, "rlc encoded value matches bytes");
-}
-
-#[test]
-fn storage_key_byte_out_of_range() {
```

Comment:
Do we remove this test because it is rlc related, so it has become obsolete now?